test/gov/agbot_apitest.sh

```diff
@@ -5,12 +5,22 @@ USERDEV_ADMIN_AUTH="userdev/userdevadmin:userdevadminpw"
 
 PREFIX="Agbot API Test:"
 
-
 echo ""
 echo -e "${PREFIX} Start testing compatibility"
 
+if [ -z ${AGBOT_SAPI_URL} ]; then
+  echo -e "\n${PREFIX} Envvar AGBOT_SAPI_URL is empty. Skip test\n"
+  exit 0
+fi
+
 COMP_RESULT=""
 
+# setup ARCH in all policies, patterns & service definition files for deploycheck test
+for in_file in "business_pol_location.json" "service_policy.json" "node_ui.json" "pattern_sloc.json" "service_location.json" "service_locgps.json"
+do
+  sed -i -e "s#__ARCH__#${ARCH}#g" in_file
```

Comment:
Got the following errors here:
sed: can't read in_file: No such file or directory
sed: can't read in_file: No such file or directory
sed: can't read in_file: No such file or directory
sed: can't read in_file: No such file or directory
sed: can't read in_file: No such file or directory
sed: can't read in_file: No such file or directory